src/SonarQube.TeamBuild.Integration/BuildVNextCoverageReportProcessor.cs

```diff
@@ -48,6 +48,13 @@ protected override bool TryGetBinaryReportFile(AnalysisConfig config, ITeamBuild
             return true; // there aren't currently any conditions under which we'd want to stop processing
         }
 
+        protected override bool TryGetTrxFile(AnalysisConfig config, ITeamBuildSettings settings, ILogger logger, out string trxFilePath)
+        {
+            trxFilePath = TrxFileReader.FindTrxFile(settings.BuildDirectory, logger);
+
```

Comment:
[cosmetic] in the previous file there is no space between the initialization and the return xD